examples/playground/panel-lite-iframe.js

```diff
@@ -0,0 +1,159 @@
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.holoviz.org/panel/1.3.0/dist/panel.min.js"></script>
+</head>
+<body>
+    <div>I am a new template</div>
+	<div id="main"></div>
+    <div id="loading-spinner" style="height:100px;width:100px">Loading...</div>
+</body>
+</html>
+`
+
+// Fix Panel when running inside an iframe srcdoc
+// See https://github.com/holoviz/panel/issues/5506#issuecomment-1784433401
+const fixPanel = `
+from panel.io.location import Location
+Location.param.pathname.regex=r"^$|[\/]|srcdoc$"
+print("Panel Location issue fixed")
+`
+
+// See https://github.com/holoviz/panel/blob/8579e5cf322604e61b95bb1e10dcc57466298df1/panel/io/convert.py#L87
+const pyoidideScript = `
+<script type="text/javascript">
+async function main() {
+    let envSpec =  [{{ env_spec }}]
+    envSpec = envSpec.filter(item => !window.envSpec.includes(item));
+    if (envSpec.length > 0) {
+        console.log("installing: ", envSpec)
+        await window.micropip.install(envSpec)
+    }
+    window.envSpec = window.envSpec.concat(envSpec);
+
+
+    code = \`{{ code }}\`
+    await window.pyodide.runPythonAsync(code);
+
+    window.addEventListener('message', handleMessage);
+  }
+  main();
+<\/script>
+`
+
+
+async function useCustomPyodideScript(){
+    command = `
+from panel.io import convert
+convert.PYODIDE_SCRIPT = \"\"\"
+${pyoidideScript}
+\"\"\"
+print("Changed the PYODIDE_SCRIPT for Panelite")
+`
+    await window.pyodide.runPythonAsync(command)
+}
+
+async function loadPyodideAndPackages(jsglobals){
+    _pyodide = await loadPyodide({
+        packages: "micropip",
+        jsglobals: jsglobals
+    })
+    window.micropip = _pyodide.pyimport("micropip");
+    window.envSpec=[
+        'https://cdn.holoviz.org/panel/wheels/bokeh-3.3.0-py3-none-any.whl',
+        'https://cdn.holoviz.org/panel/1.3.0/dist/wheels/panel-1.3.0-py3-none-any.whl',
+        'pyodide-http==0.2.1'
+    ]
+    await window.micropip.install(window.envSpec)
+    // Todo: https://github.com/holoviz/panel/issues/5811
+    window.envSpec = window.envSpec.concat(["panel", "bokeh", "datetime", "random"])
+    return _pyodide
+}
+
+async function convertToHTML(code){
+    await useCustomPyodideScript()
+
+    let command = `
+
+from io import StringIO
+from panel.io import convert
+from panel.io.convert import script_to_html
+
+code = \"\"\"
+${fixPanel}
+
+${code}
+\"\"\"
+
+def _get_html(code):
+    file = StringIO(code)
+
+    try:
+        html, _ = script_to_html(file)
+    except Exception as ex:
+        html = f"Exception: {ex}"
+        print(ex)
+
+
+    return html.replace("v0.23.4", "v0.24.1")
```

Comment:
## Todo

- [ ] Get Panel updated to use latest version of Pyodide
- [ ] Remove `.replace` when released.